### ISSUE TITLE
Flow Sensor attribute not returned and cannot troubleshoot

### ISSUE THREAD

**Issue by vdiogo** [raomin/ESPAltherma#417]
I have started using ESPAltherma today and while noting I was not retrieving some of the attribute sensors, I also cannot troubleshoot what the issue could be as I'm not receiving events on `espaltherma/log` topic:
<img width="639" alt="image" src="https://github.com/raomin/ESPAltherma/assets/24878195/20dd3ebd-450e-4ec8-8155-092edd29d8c6">

I first checked if the sensor I'm looking to return was uncommented in the code (it is):

`{0x62,9,105,2,-1,"Flow sensor (l/min)"}`

Then I checked if there was some setting in the setup.h to enable listening to the `espaltherma/log` topic (there is not).

In short I need to return the Flow sensor (l/min) so I can calculate the HP COP.

Thanks for any hint on how to troubleshoot further

**Comment by raomin** [comment]
Doing some cleanup on old issues. This one has been quiet for a long time, so I am closing it. Reopen with logs from the latest version if the flow value is still missing.
